crates/wit/src/parsed_type/vector_utils.rs

```diff
@@ -155,7 +149,7 @@ pub(crate) fn generate_vector_deserializer(
     let values_deserializer = match value_ty {
         ParsedType::Boolean(_) => {
             quote! {
-                unimplemented!("Vector of booleans is not supported")
+                unimplemented!("Vector of booleans is unsupported")
```

Comment:
i think not supported is a better message than  unsupported. tomatoe, tomato.

Reply:
Hmm, it's strange this file from an older version of sdk, now there is no such message at all.